google-cloud-storage/src/main/java/com/google/cloud/storage/transfermanager/TransferManagerConfig.java

```diff
@@ -142,24 +167,49 @@ private Builder() {
       this.qos = DefaultQos.of();
     }
 
+    /**
+     * Maximum amount of workers to be allocated to perform work in Transfer Manager
+     *
```

Comment:
It's particularly useful to include what the default value is in the javadocs for each of these setter methods.

```suggestion
     * <p><i>Default Value:</i> {@code 2 * }{@link Runtime#getRuntime()}{@code .}{@link
     * Runtime#availableProcessors() availableProcessors()}
     *
```